### PR TITLE
feat(icons): add manual focus and far away icons

### PR DESCRIPTION
**Pull request by brian-chano**
- add manual focus, near and far away icons

image

### CODE REVIEW

File: packages/icons/src/manual-focus.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M5,15H3v4a2,2,0,0,0,2,2H9V19H5ZM5,5H9V3H5A2,2,0,0,0,3,5V9H5ZM19,3H15V5h4V9h2V5A2,2,0,0,0,19,3Zm0,16H15v2h4a2,2,0,0,0,2-2V15H19Z"/>
+  <path d="M8,16V8h2.58L12,13.37,13.42,8H16v8H14V9.8L12.76,16H11.23L10,9.8V16Z" />
```
Comment (boilund) [inline packages/icons/src/manual-focus.svg:3]: Could you combine paths to one path? Guideline is [here](https://github.com/AxisCommunications/practical-react-components/blob/main/DEVELOPMENT.md#icons)

File: packages/icons/src/far-away.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (boilund) [inline packages/icons/src/far-away.svg:1]: Don't they need license? 
Please add screenshots of icons in your description in this PR.

Comment (brian-chano) [reply packages/icons/src/far-away.svg:1]: Added license and screenshots

File: packages/icons/src/manual-focus.svg (hunk 1 above)
Comment (brian-chano) [reply packages/icons/src/manual-focus.svg:3]: Done

File: packages/icons/src/far-away.svg (hunk 2 above)
Comment (danielkaxis) [reply packages/icons/src/far-away.svg:7]: This one is not based on crop_free right?

Comment (brian-chano) [reply packages/icons/src/far-away.svg:7]: Sorry, fixed

**Review by boilund — APPROVED** [review]

**Pull request merged**